engine/src/cmds.h

```diff
@@ -1738,7 +1738,7 @@ class MCGo : public MCStatement
 	Window_mode mode;
 	Boolean marked;
 	Boolean visible;
-	Boolean explicit_visibility = False;
+	MCGoStackVisibility visibilityType = kImplicit;
```

Comment:
Don't initialise here, do it in the constructor below where the other members are initialised

Reply:
Oh and you can remove the visible Boolean!